app/editor/src/components/layout/DefaultLayout.tsx

```diff
@@ -25,6 +25,10 @@ export const DefaultLayout: React.FC<ILayoutProps> = ({ name, children, ...rest
   const keycloak = useKeycloakWrapper();
   useToastError();
 
+  keycloak.instance.onTokenExpired = () => {
+    keycloak.instance.logout();
```

Comment:
Does this occur when the access token expires or the refresh token expires?

Reply:
Yes, based on the testing it will occur when the access token expires.